resources/views/app/includes/sections/modalforms.blade.php

```diff
@@ -0,0 +1,97 @@
+<!-- Modal Structure -->
+<!-- Signup Modal-->
+<div id="signup" class="modal">
+    <div class="modal-content">
+
+        <div class="row">
+            <form class="col s12">
+                <div class="row">
+                    <div class="input-field col s12">
+                        <i class="material-icons prefix">account_circle</i>
+                            <input id="last_name" type="text" class="validate" placeholder="e.g. snakeMonkeyDuh">
+                                <label for="last_name">Username</label>
+                    </div>
```

Comment:
The `<input>` and `<label>` tag should be on the same grid line as the `<i>` tag